code/game/machinery/camera/camera.dm

```diff
@@ -233,10 +244,9 @@
 	. = ..()
 
 /obj/machinery/camera/obj_break(damage_flag)
-	if(status)
+	if(status && !(flags & NODECONSTRUCT))
 		triggerCameraAlarm()
-		toggle_cam(null, FALSE)
-		wires.CutAll()

```

Comment:
TG doesn't have wires for cameras (which IMO is a good thing), but we do, so this needs to stay.